tools/update-locked-requirements

```diff
@@ -40,6 +40,18 @@ EOF
     ) | sponge "$output"
 }
 
+update_requirements_hash_file () {
+    hash_file='requirements/requirements_hash'
+    cat > $hash_file << EOF
+#
+# This file is auto-generated, please don't edit directly. In case of a merge-conflict,
+# simply delete this file and run tools/update-locked-requirements to regenerate it,
+# don't resolve merge-conflicts manually.
+#
+EOF
+    sha1sum requirements/*.txt | awk '{print $1}' | sort | sha1sum | awk '{print $1}' >> $hash_file
```

Comment:
Can you use `tools/lister.py` so that we only hash files that are checked into Git?  Otherwise random other files that might get put there will count for this.